Winleafs.Wpf/Views/Layout/LayoutDisplayUserControl.xaml.cs

```diff
@@ -16,21 +16,25 @@
 
 namespace Winleafs.Wpf.Views.Layout
 {
-    /// <summary>
-    /// Interaction logic for LayoutDisplay.xaml
-    /// </summary>
-    public partial class LayoutDisplayUserControl : UserControl
-    {
-        private static readonly SolidColorBrush _lockedBorderColor = Brushes.Red;
-        private static readonly SolidColorBrush _highLightColor = Brushes.OrangeRed;
-        private static readonly SolidColorBrush _selectedBorderColor = Brushes.LightSteelBlue;
-        private static readonly SolidColorBrush _borderColor = (SolidColorBrush)Application.Current.FindResource("NanoleafBlack");
-
-        public HashSet<int> SelectedPanelIds { get; set; }
-
-        private static readonly Random _random = new Random();
-
-        private List<DrawablePanel> _polygons;
+	/// <summary>
+	/// Interaction logic for LayoutDisplay.xaml
+	/// </summary>
+	public partial class LayoutDisplayUserControl : UserControl
+	{
+		private static readonly SolidColorBrush _lockedBorderColor = Brushes.Red;
```

Comment:
Odd how the formatting changed. Maybe it would be good for us to get more professional on the repository as well and add an .editorconfig @StijnOostdam